src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
+const LONG_TERM = 3;
+const BASEPRISE = 40;
```

Comment:
The variable `BASEPRISE` seems to be a typo. It should be `BASEPRICE` to correctly represent the base price per day.